ext/graphql_libgraphqlparser_ext/extconf.rb

```diff
@@ -1,6 +1,11 @@
 require 'mkmf'
 
-dir_config('graphql', '/usr/local/include/graphqlparser', '/usr/local/lib')
-abort 'missing libgraphqlparser' unless have_library 'graphqlparser'
+prefixes = %w(/usr /usr/local)
+prefix = prefixes.find{ |prefix| Dir["#{prefix}/lib/libgraphqlparser*"].first }
+abort 'missing libgraphqlparser' unless prefix
```

Comment:
Oh, I guess I can call `dir_config('graphql')` in that case.  That should allow it to be specified manually.